app/_src/kubernetes-ingress-controller/deployment/minikube.md

```diff
@@ -31,23 +31,55 @@ Kong is accessible. This is used to send requests into the
 Kubernetes cluster.
 
 1. Get the IP address at which you can access {{site.base_gateway}}:
-   
+   {% capture the_code %}   
+{% navtabs codeblock %}
+{% navtab kubectl %}
+```bash
+$ minikube service -n kong kong-proxy --url | head -1
+```
+{% endnavtab %}
+{% navtab Helm Charts %}
    ```bash
-   $ minikube service -n kong kong-proxy --url | head -1
-   # If installed by helm, service name would be "<release-name>-kong-proxy".
-   # minikube service <release-name>-kong-proxy --url | head -1
+   HOST=$(kubectl get svc --namespace kong kong-gateway-proxy -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Comment:
The `HOST` field will not be available unless you run `minikube tunnel` on non-Linux machines. It's worth adding a note here so that people don't get confused

Reply:
addressed